x-pack/plugin/runtime-fields/qa/rest/src/yamlRestTest/java/org/elasticsearch/xpack/runtimefields/rest/CoreTestsWithRuntimeFieldsIT.java

```diff
@@ -239,7 +239,7 @@ public void execute(ClientYamlTestExecutionContext executionContext) throws IOEx
             List<Map<String, Object>> bodies = List.of(
                 Map.ofEntries(
                     Map.entry("index_patterns", "*"),
-                    Map.entry("priority", Integer.MAX_VALUE),
+                    Map.entry("priority", Integer.MAX_VALUE - 1),
```

Comment:
@nik9000  this does the job but I am not super proud of it. I guess this is caused by our matching pattern `*` but we have no way to say "all indices but the system ones", and I am not even sure that would work either. Thoughts?

Reply:
What was wrong? The tests were passing for me without it. I don't mind it - I just don't understand it yet.

Reply:
yea this came from merging master in today.  see https://github.com/elastic/elasticsearch/pull/61776/commits/acfaccb715f714ebe92b478f9dd3cdfbe7b40228

Reply:
Gotcha. It makes sense. It's kind of a shame we don't override the internal templates too to prove that all of those also work with runtime fields but its cool.